src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -0,0 +1,246 @@
+// @flow
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import { FormattedMessage } from 'react-intl';
+import TetherComponent from 'react-tether';
+import Avatar from '../Avatar';
+import Media from '../../../../components/media';
+import { MenuItem } from '../../../../components/menu';
+import IconTrash from '../../../../icons/general/IconTrash';
+import IconPencil from '../../../../icons/general/IconPencil';
+import { ACTIVITY_TARGETS } from '../../../common/interactionTargets';
+import DeleteConfirmation from '../common/delete-confirmation';
+import ActivityTimestamp from '../common/activity-timestamp';
+import UserLink from '../common/user-link';
+import ActivityError from '../common/activity-error';
+import ActivityMessage from '../common/activity-message';
+import AnnotationActivityItem from './AnnotationActivityItem';
+import CommentForm from '../comment-form';
```

Comment:
Given the number of imports in this file, it seems like it needs to be broken into several smaller components.